docs/extend/packaging-zos-extensions.md

```diff
@@ -0,0 +1,94 @@
+
+The manifest file contains the following properties:
+
+- **`name`**: (Required) Defines a short, computer-readable name of the component. This component name will be used as directory name after it is installed. The allowed characters in the name are alphabets, numbers, hyphen (`-`) and underscore (`_`). For example, `explorer-jes` is a valid extension name.
+- **`id`**: (Optional) Defines a long, computer-readable identifier of the component. If the component is hosted as one of the projects in [Open Mainframe Project](https://www.openmainframeproject.org/), the identifier also matches the component path in the Zowe Artifactory. For example, `org.zowe.explorer-jes` is a valid identifier. You can locate the component's official releases by looking into the `libs-release-local/org/zowe/explorer-jes/` directory in the [Zowe Artifactory](https://zowe.jfrog.io/ui/repos/tree/General/libs-release-local%2Forg%2Fzowe%2Fexplorer-jes).
+- **`version`**: (Optional but recommended) This is the current version of the component without the prefix of `v`. For example, `1.0.4` is a valid `version` value.
+- **`title`**: (Optional) Defines a short human-readable name for this component. This value will also be used as the default title for API Catalog tile, or App Framework plug-in title. For example, `JES Explorer` is a valid `title` for the `explorer-jes` component.
+- **`description`**: (Optional) Defines a long human-readable description of this component. There is no restriction on what you can put in the field.
+- **`license`**: (Optional but recommended) Defines the license code of the component. For example, Zowe core components have `EPL-2.0` value in this field.
+- **`build`**: (Optional but strongly recommended) Defines the build information of the current package, including git commit hash, and so on. When Zowe core components define manifest file, these fields are left as template variables. The template will be updated when a publishable package is created. It supports the following subfields:
+  * **`branch`**: It tells the user which branch this package is built from.
+  * **`number`**: You may create multiple packages in the same branch. This is the sequential number of the current package.
+  * **`commitHash`**: This is the commit hash of the package that can be used to match the exact source code in the repository. Zowe core components usually use `git rev-parse --verify HEAD` to retrieve the commit hash.
+  * **`timestamp`**: This is the UNIX timestamp when the package is created.
+- **`commands`**: This field is a high level of several subfields to define actions that should be taken when the component is installed, configured, started, or tested. All subfields are optional and usually should point to a USS command or script.
```

Comment:
"`This field is a high level of several subfields to define actions that should be taken when the component is installed,..`" should this be "`This field is a high level command with several subfields to define actions that should be taken when the component is installed,`" ?

Reply:
Thanks a lot Ashley for all the fixes. Sorry this is confusing, what I was trying to say is `commands` itself is not a recognized command can be handled by the installer, it must be combined with a subfield, for example `commands.install`. I can understand your fixes without problem.

Reply:
Changed the description to
`This defines actions that should be taken when the component is installed, configured, started, or tested. You must issue this command with one subfield as listed below. For example, `commands.install`. All subfields are optional and usually should point to a USS command or script.`

Reply:
This sounds great! Thank you Ashley!